tests/functional/test_import.py

```diff
@@ -28,15 +17,19 @@ def test_service_import_negative(sdk_client_fs: ADCMClient, path):
     3. Bind service from cluster with export to cluster with import
     4. Expect backend error because incorrect version for import
     """
-    bundle = sdk_client_fs.upload_from_fs(path + '/export')
-    bundle_import = sdk_client_fs.upload_from_fs(path + '/import')
-    cluster = bundle.cluster_create("test")
-    service = cluster.service_add(name="hadoop")
-    cluster_import = bundle_import.cluster_create("cluster_import")
-    cluster_import.bind(cluster)
+    with allure.step('Create cluster with def export'):
+        bundle = sdk_client_fs.upload_from_fs(path + '/export')
+        cluster = bundle.cluster_create("test")
+        service = cluster.service_add(name="hadoop")
+    with allure.step('Create cluster with def import'):
+        bundle_import = sdk_client_fs.upload_from_fs(path + '/import')
+        cluster_import = bundle_import.cluster_create("cluster_import")
+    with allure.step('Bind service from cluster with export to cluster with import'):
+        cluster_import.bind(cluster)
     with pytest.raises(coreapi.exceptions.ErrorMessage) as e:
         cluster_import.bind(service)
-    err.BIND_ERROR.equal(e)
+    with allure.step('Expect backend error because incorrect version for import'):
+        err.BIND_ERROR.equal(e)
```

Comment:
```suggestion
        assert err.BIND_ERROR == e
```

Reply:
View in all similar places

Reply:
https://arenadata.atlassian.net/browse/ADCM-1581

Reply:
Pls revert. Didn't know it was a custom `equal`